src/test/java/HomePageTest.java

```diff
@@ -7,26 +8,24 @@
 public class HomePageTest extends BaseTest{
 
     @Test
-    public void SignUpSuccess () throws InterruptedException {
-
-        driver.get("https://kwidos.com/auth/register/contractor");
+    public void SignUpSuccess () {
         List<Character> charSet = new ArrayList<>(Arrays.asList('a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', '1', '2'));
         Collections.shuffle(charSet, new Random());
         String rand="";
         for (int i = 0; i < charSet.size(); i++) {
             rand=rand+charSet.get(i);
```

Comment:
it is not good solution because you have finite number of combinations. It would be better to use System.currentTimeMillis() to generate unique number and add it to email name